README.md

```diff
@@ -0,0 +1,10 @@
+[![Documentation Status](https://readthedocs.org/projects/tron-wiki/badge/?version=latest)](http://tron-wiki.readthedocs.io/en/latest/?badge=latest)
+                
+# TRON Wiki
+
+## Getting Started
+The documentation is currently hosted on http://wiki.tron.network
+
+## Contributing
```

Comment:
@ruchern can we please remove those two empty sections for now so we can publish this. you can then work on those at later time.

Reply:
Sorry, I will take this out now.